### ISSUE TITLE
Vimeo Link

### ISSUE THREAD

**Issue by jessican01** [ASL-LEX/SignLab#85]
**Is your feature request related to a problem? Please describe.**
It is confusing that I need a Vimeo link for the video option, but not the other options.

**Describe the solution you'd like**
 It would be nice for them to all be similar (upload from the database rather than having to upload the videos to Vimeo).

**Comment by cbolles** [comment]
@jessican01   That sounds good. Would you like the expected workflow to be something where you upload information on the videos you want to support (Codes, English text to search over) then upload the videos to be stored on SignLab?

**Comment by jessican01** [comment]
Yes, that would be great! That way I won't have to upload it to vimeo. Thanks!
